### PR TITLE
hw 27 solution

### PR DESCRIPTION
**Pull request by programmer28**

### CODE REVIEW

File: src/main/java/com/dev/cinema/model/CinemaHall.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_hall")
+public class CinemaHall {
+
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/CinemaHall.java]: @programmer28 please remove this line

File: src/main/java/com/dev/cinema/model/MovieSession.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/MovieSession.java]: @programmer28 please remove this line

File: src/main/java/com/dev/cinema/model/CinemaHall.java (hunk 1 above)
Comment (programmer28) [reply src/main/java/com/dev/cinema/model/CinemaHall.java]: I`ve removed it

File: src/main/java/com/dev/cinema/model/MovieSession.java (hunk 2 above)
Comment (programmer28) [reply src/main/java/com/dev/cinema/model/MovieSession.java]: I`ve removed it

**Pull request merged**